pandas/tests/io/test_parquet.py

```diff
@@ -368,6 +368,41 @@ def test_multiindex_with_columns(self, pa_ge_070):
             check_round_trip(df, engine, read_kwargs={'columns': ['A', 'B']},
                              expected=df[['A', 'B']])
 
+    def test_write_ignoring_index(self, engine):
+        # ENH 20768
+        # Ensure index=False omits the index from the written Parquet file.
+        df = pd.DataFrame({'a': [1, 2, 3], 'b': ['q', 'r', 's']})
+
+        # Because we're dropping the index, we expect the loaded dataframe to
+        # have the default integer index.
+        expected = df.reset_index(drop=True)
+
+        check_round_trip(df, engine, write_kwargs={'index': False},
```

Comment:
reason for the failing test with `RuntimeError: Compression 'snappy' not available` is that `check_round_trip` normally sets `write_kwargs` to `{'compression': None}` is it is not specified (because `fastparquet` does not use the snappy compression). So to solve this you can add `'compression': None` here to the dict as well.

Reply:
Great, thanks!